docs/3_development/1_development-flow/3_review.md

```diff
@@ -27,6 +27,262 @@ Walkthroughs is peer programming for reviews, if possible/required, ask for a wa
+
+### Speed of the code review
+
+#### Why should code reviews be fast
+
+Because the team is the most important! If code reviews are slow, few things will happen for sure:
+
+- the author will start working on something else while waiting, and keep forgetting the context
+- the author will finally get the reviewer’s feedback, but now he is in the middle of something else. Days will creep in.
+- then the author will need to refresh his memory and address changes, making the reviewer forget about the context.
+- the team will lose time and small cracks of frustration and unhappiness will appear in the team.
+
+This is only worse if more reviewers are assigned. We will waste significant time on this context switching. It is a very inefficient process!
+
+#### How fast should code reviews be
+
+If we are not in the middle of a focused task, we should do a code review shortly after it comes in.
+
+One business day is the maximum time it should take to respond to a code review request (i.e., the first thing the next morning).
+
+Following these guidelines means that a typical code change request should get multiple rounds of review (if needed) within a single day.
+
+📢 If we are in the middle of a focused task, such as writing code, we shouldn’t interrupt ourselves to do a code review. Instead, we should wait for a breakpoint in our work to respond to code change request.
+
+#### Fast responses
+
+When we talk about fast code reviews, we are mainly talking about the response time. We should take time when reviewing the actual code change request. We need to fully understand the code. Ideally, the whole process should be fast. Remember, the code change doesn’t have to be perfect!
+
+📢 If we are too busy to do a full review, we can still send a quick response that lets the developer know when we will get to it. Or we can suggest another reviewer! We should delegate, and don’t allow the code change request to sour for days.
+
+#### Large code change requests
+
+Are unacceptable and reviewers have the right to break the fast code review rule. Usually the way-to-go is to ask the author to split the code change into smaller requests (all branches should be branched from single/main branch for that feature). This way, all requests can be reviewed in a single day and when they're merged, the feature is complete and large code change is more-or-less already reviewed, so we only need to do a simple final check.
+
+### How to write code review comments
+
+#### 3 magic questions we should always ask ourselves
+
+Always:
+
+1. is it kind
+2. is it an absolute truth
+3. is it necessary
+
+#### Courtesy
+
+In general, it is important to be [courteous and respectful](https://chromium.googlesource.com/chromium/src/+/master/docs/cr_respect.md) while also being very clear and helpful to the developer whose code we are reviewing. One way to do this is to be sure that we are always making comments about the code and never making comments about the developer.
```

Comment:
maybe drop the link as it relates to Google culture

Reply:
but is it that tightly coupled with it? because it's just 8 DOs and 4 DON'Ts which seem pretty generic?

Reply:
In any case it should.prpbably be permalink 🙂

Reply:
agreed, but i think this is the only way it's usable (they use it too) :sadge:

Reply:
i mean, i guess i can extract those dos and donts to this doc if we don't want to depend on Google haha